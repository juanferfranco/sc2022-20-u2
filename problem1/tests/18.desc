crea varios eventos y los destruye del Ãºltimo al primero creados