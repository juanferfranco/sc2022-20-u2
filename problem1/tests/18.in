add
ev1
add
ev2
add
ev3
ask
rm
ev3
ask
rm
ev2
ask
rm
ev1
ask
exit
