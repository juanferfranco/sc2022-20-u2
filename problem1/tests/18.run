./main < tests/18.in
