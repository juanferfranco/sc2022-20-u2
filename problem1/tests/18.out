ev1
ev2
ev3
ev1
ev2
ev1
empty
